Components/Buttons/Buttons.css

```diff
@@ -0,0 +1,31 @@
+.btn {
+  border: none;
+  padding: 0.6rem 1rem;
+  color: white;
+  border-radius: 0.25rem;
+  font-weight: 600;
+  letter-spacing: 0.5px;
+  background: grey;
+  font-family: "roboto";
+  font-size: 1rem;
+  height: 3rem;
+  box-shadow: rgb(0 0 0 / 17%) 0px 4px 9px;
+}
+
+.icon {
+  color: var(--primary-color) !important;
```

Comment:
i remember Vikram suggesting to not use `!important` so do check this once

Reply:
I have enforced a diffrent font color all across and it was interfering with component font color

Reply:
yeah got it